src/style.css

```diff
@@ -0,0 +1,137 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(.\fonts\Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  src: url(.\fonts\Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.search-field {
+  display: flex;
+  align-items: center;
+  margin-top: 20px;
+}
+
+.search-field:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.search-icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  margin-left: 26px;
+  z-index: 20;
+}
+
+.label-input {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.label-input::before {
+  position: absolute;
+  content: "";
+  background-image: url(./images/Search.svg);
+  width: 19px;
+  height: 19px;
+  z-index: 100;
+  top: 50%;
+  transform: translateY(-50%);
+  left: 26px;
+}
+
+.search-input {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 62px;
+  height: 70px;
+  width: 100%;
+  font-family: Avenir, sans-serif;
```

Comment:
Remove `font-family` here, cause you have this property for all elements in `body` selector.